### DISCUSSION TITLE
Unable to open Draft release

### DISCUSSION THREAD

**Discussion by AmauriC** [community/community#124794] [Mobile]
"Something went wrong, try again"

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.162.0 (130645053)
Device: iPhone 15 Pro Max (iOS 17.4.1)
</details>

![image](https://github.com/community/community/assets/8764074/39b1b9af-61b0-4601-bc17-e6697fbaacf7)
![image](https://github.com/community/community/assets/8764074/1a4c7029-8175-4f6b-8f30-ebd447aa86a7)

**Comment by alcere** [comment]
Hi @AmauriC! Thank you for bringing this issue to our attention. I have created an internal issue, and the iOS team will investigate and work on a fix. We apologize for any troubles this may have caused you!
